docs/remote_inference_blueprints/standard_blueprints/README.md

```diff
@@ -0,0 +1,51 @@
+# Model Blueprints for Vector Search
+
+OpenSearch provides two approaches for implementing embedding models, depending on your needs:
+
+## Standard Blueprints (Using ML Inference Processor)
+Recommended for new implementations
+
+These blueprints use the ML inference processor to handle input/output mapping, offering:
+
+- Simpler implementation
+- Direct model registration 
+- Flexible input/output mapping through the processor
+
+[List of available models]:
```

Comment:
I have several more non-embedding models here, can you add them? https://github.com/opensearch-project/dashboards-flow-framework/blob/main/documentation/models.md

Reply:
this PR is to add blueprint for vector search. Also for non-embedding model, they don't have pre and post processing function. we still can use the legacy blueprints we had.